_scripts/_hotReloadLocalesScript.js

```diff
@@ -0,0 +1,18 @@
+const websocket = new WebSocket('ws://localhost:9080/ws')
+
+websocket.onmessage = (event) => {
+  const message = JSON.parse(event.data)
+
+  if (message.type === 'freetube-locale-update') {
+    const i18n = document.getElementById('app').__vue__.$i18n
```

Comment:
There's so much I don't know. This is so cool. Side tip I found from looking this up for @ChunkyProgrammer: the Vue 3 equivalent is apparently [`__vueParentComponent.ctx`](https://stackoverflow.com/a/64429013).